persistence/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraPersistentEntityRegistryImpl.scala

```diff
@@ -114,8 +105,19 @@ private[lagom] class PersistentEntityRegistryImpl @Inject() (system: ActorSystem
   private def entityTypeName(entityClass: Class[_]): String = Logging.simpleName(entityClass)
 
   override def eventStream[Event <: AggregateEvent[Event]](
-    aggregateTag: AggregateEventTag[Event],
-    fromOffset:   Optional[UUID]
+    fromOffset:   Optional[Long],
```

Comment:
Perhaps introduce a new `Offset` type instead, that could carry a UUID or a Long (or whatever other thing that other implementations might use)?